### PR TITLE
Body converter should consume request body

### PR DESCRIPTION
**Pull request by alekitto**

### CODE REVIEW

File: src/Requester/Decorator/BodyConverterDecorator.php
Comment (massimilianobraglia) [inline src/Requester/Decorator/BodyConverterDecorator.php:77]: `: string` as return type

**Review by massimilianobraglia — COMMENTED** [review]
Nice one, @alekitto! Just left a really minor comment

File: src/Requester/Decorator/BodyConverterDecorator.php
Comment (alekitto) [reply src/Requester/Decorator/BodyConverterDecorator.php:77]: Not really always a string: if `$length` is null it could be a resource.

Anyway, the coding standard has a rule for that and should raise an error if a type hinting is missing. No need to check them manually anymore 🙂

**Comment by alekitto** [comment]
This is relative to a bug discovered yesterday with @rpanfili.
I'll wait for his feedback of this final version before merging and release it.

File: src/Requester/Decorator/BodyConverterDecorator.php
Comment (massimilianobraglia) [reply src/Requester/Decorator/BodyConverterDecorator.php:77]: Ok, I'll resolve this one then

**Review by rpanfili — APPROVED** [review]
lgtm! 👍

**Comment by alekitto** [comment]
Released in [0.3.1](https://github.com/solid-o/atlante-php/releases/tag/0.3.1)

**Pull request merged**
